src/App.tsx

```diff
@@ -1,10 +1,80 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect,
+  useState,
+  useMemo,
+  useCallback,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Sort } from './utils/Sort';
+import { ErrorType } from './utils/ErrorType';
+import { TodosList } from './components/TodosList';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error';
 
-const USER_ID = 0;
+const USER_ID = 10360;
```

Comment:
Consider setting USER_ID back to 0 or a valid user ID to avoid possible issues with the API calls.